Libplanet/Address.cs

```diff
@@ -320,6 +320,13 @@ private static byte[] DeriveAddress(string hex)
                 throw new ArgumentNullException(nameof(hex));
             }
 
+            int pos = hex.IndexOf('x');
+
+            if (pos >= 0)
+            {
+                hex = hex.Remove(0, pos + 1);
+            }
+
```

Comment:
I guess, it would be better if we have sanity check before this removal because we don't have to consider about other case than length is 42.

```suggestion
            if (hex.Length == 42)
            {
                int pos = hex.IndexOf('x');
                if (pos >= 0)
                {
                    hex = hex.Remove(0, pos + 1);
                }
            }
```

Reply:
Eh, I think we need to explicitly check if the prefix is `0x`. 😶
This matches regex `.+x`. Although the if statement below would only let `.x`, this is still not the same as removing `0x`.

Reply:
Agree for that. I think it's some kind of room(?") for first time contributer😅. we may need to follow for that soon.